src/sentry/api/issue_search.py

```diff
@@ -92,6 +94,27 @@ def convert_status_value(value, projects, user, environments):
     return parsed
 
 
+def convert_category_value(value, projects, user, environments):
+    if features.has("projects:performance-issue-details-backend", projects[0]):
+        for category in value:
+            for member in GroupCategory:
+                if category.upper() == member.name:
+                    issue_types = GROUP_CATEGORY_TO_TYPES.get(member, [])
+                    return [type.value for type in issue_types]
```

Comment:
`category = getattr(GroupType, category.upper(), None)` 

We might end up needing a map of these names -> UI names at some point. I'm sure we'll end up changing them and not want to rename the enums. Can do that when it happens though